specification/common-types/resource-management/v6/types.json

```diff
@@ -158,59 +224,47 @@
         },
         "isDataAction": {
           "type": "boolean",
-          "description": "Whether the operation applies to data-plane. This is \"true\" for data-plane operations and \"false\" for ARM/control-plane operations.",
+          "description": "Whether the operation applies to data-plane. This is \"true\" for data-plane operations and \"false\" for Azure Resource Manager/control-plane operations.",
           "readOnly": true
         },
         "display": {
-          "type": "object",
-          "description": "Localized display information for this particular operation.",
-          "properties": {
-            "provider": {
-              "type": "string",
-              "description": "The localized friendly form of the resource provider name, e.g. \"Microsoft Monitoring Insights\" or \"Microsoft Compute\".",
-              "readOnly": true
-            },
-            "resource": {
-              "type": "string",
-              "description": "The localized friendly name of the resource type related to this operation. E.g. \"Virtual Machines\" or \"Job Schedule Collections\".",
-              "readOnly": true
-            },
-            "operation": {
-              "type": "string",
-              "description": "The concise, localized friendly name for the operation; suitable for dropdowns. E.g. \"Create or Update Virtual Machine\", \"Restart Virtual Machine\".",
-              "readOnly": true
-            },
-            "description": {
-              "type": "string",
-              "description": "The short, localized friendly description of the operation; suitable for tool tips and detailed views.",
-              "readOnly": true
-            }
-          }
+          "$ref": "#/definitions/OperationDisplay",
+          "description": "Localized display information for this particular operation."
         },
         "origin": {
-          "type": "string",
+          "$ref": "#/definitions/Origin",
           "description": "The intended executor of the operation; as in Resource Based Access Control (RBAC) and audit logs UX. Default value is \"user,system\"",
-          "enum": [
-            "user",
-            "system",
-            "user,system"
-          ],
-          "x-ms-enum": {
-            "name": "Origin",
-            "modelAsString": true
-          },
           "readOnly": true
         },
         "actionType": {
+          "$ref": "#/definitions/ActionType",
+          "description": "Extensible enum. Indicates the action type. \"Internal\" refers to actions that are for internal only APIs.",
+          "readOnly": true
+        }
+      }
+    },
+    "OperationDisplay": {
+      "type": "object",
+      "description": "Localized display information for and operation.",
```

Comment:
>and [](http://example.com/codeflow?start=56&length=3)

asked on v5 also: this 'and operation' description is messed up, did we mean to say 'an operation'?